src/Gui/ExpressionCompleter.cpp

```diff
@@ -562,6 +570,14 @@ void ExpressionLineEdit::setNoProperty(bool enabled) {
         completer->setNoProperty(enabled);
 }
 
+void ExpressionLineEdit::setMatchExact(bool enabled) {
+    matchExact = enabled;
+#if QT_VERSION>=QT_VERSION_CHECK(5,2,0)
+    if (completer)
+        completer->setFilterMode(matchExact ? Qt::MatchExactly : Qt::MatchContains);
```

Comment:
According to https://doc.qt.io/qt-5/qcompleter.html#filterMode-prop Qt::MatchExactly is not supported. It must be replaced with Qt::MatchStartsWith which is the default